libs/domains/services/feature/src/lib/service-details/service-details.tsx

```diff
@@ -0,0 +1,221 @@
+                  {gitRepository.url && gitRepository.name && (
+                    <>
+                      <Dt>Repository:</Dt>
+                      <Dd>
+                        <a href={gitRepository.url} target="_blank" rel="noopener noreferrer">
+                          <Badge variant="surface" size="xs" className="gap-2">
+                            <Icon
+                              name={gitRepository.url.includes('//github') ? IconEnum.GITHUB : IconEnum.GITLAB}
+                              height={14}
+                              width={14}
+                            />
+                            <Truncate text={gitRepository.name} truncateLimit={20} />
+                          </Badge>
+                        </a>
+                      </Dd>
+                    </>
+                  )}
+                  {gitRepository.branch && (
+                    <>
+                      <Dt>Branch:</Dt>
+                      <Dd>
+                        <Badge variant="surface" size="xs" className="gap-2">
+                          <Icon name={IconAwesomeEnum.CODE_BRANCH} height={14} width={14} />
+                          <Truncate text={gitRepository.branch} truncateLimit={20} />
+                        </Badge>
+                      </Dd>
+                    </>
+                  )}
+                  <Dt>Commit:</Dt>
+                  <Dd>
+                    <LastCommit gitRepository={gitRepository} serviceId={serviceId} serviceType={service.serviceType} />
+                  </Dd>
+                </Dl>
+              )
+            )
+          })
+          .otherwise(() => undefined)}
+        {containerImage && (
+          <Dl>
+            <Dt>Image name:</Dt>
+            <Dd>{containerImage.image_name}</Dd>
+            <Dt>Tag:</Dt>
+            <Dd>{containerImage.tag}</Dd>
+          </Dl>
+        )}
+        {databaseSource && (
+          <Dl>
+            <Dt>Type:</Dt>
+            <Dd>
+              <Badge size="xs" variant="surface">
+                <Icon name={databaseSource.type} className="mr-1" height={10} width={10} />
+                {databaseSource.version}
+              </Badge>
+            </Dd>
+            <Dt>Mode:</Dt>
+            <Dd className="capitalize">{databaseSource.mode.toLowerCase()}</Dd>
+            <Dt>Accessibility:</Dt>
+            <Dd className="capitalize">{databaseSource.accessibility?.toLowerCase()}</Dd>
+          </Dl>
+        )}
+      </Section>
+      {resources && (
+        <>
+          <hr />
+          <Section className={sectionClassName}>
+            <div className="flex flex-row justify-between">
+              {/* XXX: Should be Heading, typography & design wanted */}
+              <span className="flex items-center gap-1 text-neutral-400 font-medium">
+                Resources
+                <Tooltip content="This is based on the configuration of your service">
+                  <span className="flex items-center">
```

Comment:
Ugly but visually better

```suggestion
                  <span className="flex items-center relative top-[1px]">
```

Reply:
This is not necessary in the latest version